keps/sig-cloud-provider/2025-extend-konnectivity-for-both-directions/kep.yaml

```diff
@@ -0,0 +1,50 @@
+title: Extending Apiserver Network Proxy to handle traffic originated from Node network
+kep-number: 2025
+authors:
+  - "@irozzo-1A"
+  - "@youssefazrak"
+owning-sig: sig-cloud-provider
+participating-sigs:
+  - sig-network
+status: provisional
```

Comment:
Hi there, 1.21 Enhancements Lead here. 

Please make sure to change the status to `implementable` to meet one of the requirements for all KEP tracking for the release.

Reply:
Hi @annajung, it's done.